templates/register.html

```diff
@@ -1,20 +1,20 @@
-<html>
+{% extends "layout.html" %}
 
-<body>
-<head>
-    <title>BlueberryBnB Registration</title>
-</head>
-<h1>Registration page</h1> 
+
+
+
+{% block title %}
+    Register
+{% endblock %}
+
+{% block main %}
+
+<h1>Register</h1> 
 
 <form action = "/register" method = "POST">
     <input type = "email" name = "email" placeholder = "email address"/>
```

Comment:
Just needs labels. All good otherwise

Reply:
Labels now added to register.html, alongside other changes for consistency. See new commit message.

Reply:
App.py was also updated to include the routing for getting the page add_property.html (for testing purposes!)